web/packages/shared/components/DesktopSession/Withholder.tsx

```diff
@@ -42,22 +42,23 @@ export class Withholder {
    * The internal array of keystrokes that are currently
    * being withheld.
    */
-  private withheldKeys: Array<WithheldKeyboardEventHandler> = [];
+  private withheldKeys: Array<WithheldInputEventHandler> = [];
```

Comment:
?
```suggestion
  private withheldInputs: Array<WithheldInputEventHandler> = [];
```
Also the comment should be adjusted.